packages/react-devtools-shared/src/utils.js

```diff
@@ -101,6 +101,17 @@ export function getAllEnumerableKeys(
   return keys;
 }
 
+// Mirror https://github.com/facebook/react/blob/7c21bf72ace77094fd1910cc350a548287ef8350/packages/shared/getComponentName.js#L27-L37
+export function getWrappedDisplayName(
+  outerType: mixed,
+  innerType: any,
+  wrapperName: string,
+  fallbackName?: string,
+): string {
+  const functionName = getDisplayName(innerType, fallbackName);
+  return (outerType: any).displayName || `${wrapperName}(${functionName})`;
+}
+
```

Comment:
Nit: Could just inline this

Reply:
Makes sense. It actually save potentially wasted work if `outerType.displayName` is set.